### PR TITLE
Adding missing docstrings to Bio.motifs - issue 1203

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #1203 ...

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/motifs/jaspar/__init__.py [hunk 1]
```diff
@@ -313,6 +315,11 @@ def _read_jaspar(handle):
 
 
 def calculate_pseudocounts(motif):
+    """Calculate pseudocounts, for compatibility with Perl TFBS modules.
```
Comment (mdehoon) [inline Bio/motifs/jaspar/__init__.py]: I think we can drop "for compatibility with Perl TFBS modules". This is a commonly used method to calculate pseudocounts; it's not specific to the Perl TFBS modules.

**Comment by peterjc** [comment]
If you drop the ``for compatibility with Perl TFBS modules`` bit as requested by Michael, then next time the TravisCI checks should be fine - see e8b0520aa9736bc30c55e9eb8adc82f60c0780ac and preceding commits which updated things for the new checks in the latest flake8 release.

File: Bio/motifs/jaspar/__init__.py (hunk 1 above)
Comment (svalqui) [reply Bio/motifs/jaspar/__init__.py]: Ok, is removed now, Thank You.

**Comment by svalqui** [comment]
@peterjc , ok, is amended now; I'll update my pyflakes, my version is behind; Thank You.

**Comment by mdehoon** [comment]
Thanks!

**Pull request merged**